core/base-service/base.spec.js

```diff
@@ -101,27 +103,21 @@ describe('BaseService', function() {
       )
     })
 
-    it('Should throw if route is not overridden', async function() {
-      try {
-        await BaseService.invoke({}, {}, {})
```

Comment:
Though I personally prefer using `async/await`, it's more of a stylistic opinion and not anything i feel strongly about